Sources/GDSCommon/Utilities/GDSLocalisedString.swift

```diff
@@ -33,6 +57,15 @@ extension GDSLocalisedString: ExpressibleByStringLiteral {
         stringKey = value
         variableKeys = []
         bundle = .main
+        self.attributes = nil
+    }
+    
+    public init(stringLiteral value: StringLiteralType,
+                attributes: Attributes) {
+        stringKey = value
```

Comment:
Wonder if it might be better to call init in the class here as its the only part of the code that is not covered by tests, could maybe try this and see it increases the coverage

`self.init(stringKey: value, [], bundle: .main, attributes: attributes)`